<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaInfoTipoServicio</class>
 <widget class="QDialog" name="VentanaInfoTipoServicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informaci√≥n del tipo de servicio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../recursos/qtRecursosIconos.qrc">
    <normaloff>:/iconos/iconos/icono-app-dgtic.png</normaloff>:/iconos/iconos/icono-app-dgtic.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#VentanaInfoTipoServicio {
	background-color: #fff;
}

#frameGeneralInfoTipoServicio .QPushButton{
	color: #ffffff;
	background-color: #417fd2;
	border: 1px solid rgb(0, 0, 0);
	border-radius: 6px;
	padding-bottom: 2px;
	font: 8.5pt &quot;Segoe UI Black&quot;;
}

#frameGeneralInfoTipoServicio .QPushButton::hover{
	background-color: #5c8bc9;
}

#frameGeneralInfoTipoServicio .QPushButton::pressed{
	background-color: #063c85;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameGeneralInfoTipoServicio">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
      <item>
       <widget class="QFrame" name="frameInfoTipoServicio">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>371</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="hlInfoTipoServicio">
          <item>
           <widget class="QLabel" name="labelInfoNombreTipoServicio">
            <property name="text">
             <string>Nombre del tipo de servicio:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputInfoNombreTipoServicio"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameAccionesInfoTipoServicio">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>371</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="hlBotonesAccionTipoServicio">
          <item>
           <widget class="QPushButton" name="botonAcualizarInfoTipoServicio">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Actualizar info</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonEliminarTipoServicio">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonCancelarInfoTipoServicio">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Cancelar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../recursos/qtRecursosIconos.qrc"/>
 </resources>
 <connections/>
</ui>
